src/test/java/org/mule/module/xml/xpath/XPathNamespaceKeepingSaxonNewlinesTestCase.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.xml.xpath;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+import org.mule.module.xml.XmlTestCase;
+
+import java.io.InputStream;
+import java.util.List;
+
+import org.junit.Test;
+
+public class XPathNamespaceKeepingSaxonNewlinesTestCase extends XmlTestCase {
+
+  @Override
+  protected void doSetUpBeforeMuleContextCreation() throws Exception {
+    // Testing kept newlines. Setting property accordingly
+    super.doSetUpBeforeMuleContextCreation();
+    System.setProperty("xmlModuleKeepNewlinesConfig", "true");
```

Comment:
shouldn't this be a parameterized test instead in which you try this against both options?